build.gradle

```diff
@@ -1042,17 +997,17 @@ project('spring-xd-ui') {
 	task setupUI(dependsOn: ['grunt_build', 'build']) { description = "Build the Admin UI using Grunt" }
 
 	tasks['build'].shouldRunAfter 'grunt_build'
-	tasks['compileJava'].dependsOn(['clean'])
 	grunt_build.shouldRunAfter 'installGrunt'
 	grunt_teste2e.shouldRunAfter 'installGrunt'
 	installGrunt.shouldRunAfter 'npmInstall'
 
 	task cleanUI(dependsOn: [
-			'clean',
 			'cleanDist',
 			'cleanNodeModules',
 			'cleanBowerFiles'
 	]) { description = "Clean the Admin UI dist, node modules and bower files" }
+	
+	//tasks['clean'].dependsOn(['cleanUI']) // 'dist' files are checked into VCS for now, don't clean
```

Comment:
See my comment above.

Line 1010 is commented out because artifacts are checked into source control. If they were not, we'd de-comment it